modules/understanding-upgrade-channels.adoc

```diff
@@ -14,10 +14,112 @@ recommending the appropriate upgrade versions to your cluster. Upgrade channels
 separate upgrade strategies and also are used to control the cadence of updates.
 Channels are tied to a minor version of {product-title}. For instance,
 {product-title} {product-version} channels will never include an upgrade to a
-4.3 release. This ensures administrators make an explicit decision to upgrade to
+4.5 release. This ensures administrators make an explicit decision to upgrade to
 the next minor version of {product-title}. Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+{product-title} {product-version}, which includes the upgrade from the previous 4.3
```

Comment:
and with this, we'll also want y-1 variables so we don't have to remember to manually bump the 4.3 here.  Obviously not a blocker for this PR.

Reply:
@kalexand-rh agree

Reply:
@vikram-redhat, do you have thoughts of the feasibility of adding `n+1` and `n-1` variables?

Reply:
I tested this today and couldn't find a viable way to automate the n+1 and n-1 in AsciiDoc. So we will need to manually do this.

A good thing with manual updates is that we have a process to scan the docs for just such variable name changes right before a release so it should normally get caught with that (and have a person review the change rather than it being automatic,  like with cases where we jump versions).